core-services/image-mirroring/openshift/mapping_origin_4_14

```diff
@@ -216,6 +216,7 @@ registry.ci.openshift.org/origin/4.14:metering-helm quay.io/openshift/origin-met
 registry.ci.openshift.org/origin/4.14:metering-hive quay.io/openshift/origin-metering-hive:4.14 quay.io/openshift/origin-metering-hive:4.14.0 quay.io/openshift/origin-metering-hive:latest
 registry.ci.openshift.org/origin/4.14:metering-presto quay.io/openshift/origin-metering-presto:4.14 quay.io/openshift/origin-metering-presto:4.14.0 quay.io/openshift/origin-metering-presto:latest
 registry.ci.openshift.org/origin/4.14:metering-reporting-operator quay.io/openshift/origin-metering-reporting-operator:4.14 quay.io/openshift/origin-metering-reporting-operator:4.14.0 quay.io/openshift/origin-metering-reporting-operator:latest
+registry.ci.openshift.org/origin/4.14:metrics-server quay.io/openshift/origin-metrics-server:4.14 quay.io/openshift/origin-metrics-server:4.14.0 quay.io/openshift/origin-metrics-server:latest
```

Comment:
this is the upstream k8s implementation, right?  Can we include kube in the name?

some example existing image names:
kube-state-metrics
kube-storage-version-migrator
etc...

and
kubernetes-tests

so `kube-` is the more common convention currently, and would group nicely w/ those existing images, though `kubernetes-metrics-server` would more precisely align to the upstream+repo name.

Reply:
This is addressed, can you review again? :)